witness/src/Dscp/Witness/CLI.hs

```diff
@@ -104,7 +105,7 @@ committeeParamsParser =
          help "Committee participant index. In event of secret key file \
               \generation, will be used to derive the secret.")
 
-    commSecretParser = CommitteeSecret <$> strOption
+    commSecretParser = leftToPanic . mkCommitteeSecret <$> strOption
```

Comment:
Again, why panic? this is a parser which can fail (with `empty`), so if an empty string is encountered, it should fail properly.